src/CoreBundle/DataAccess/MenuItemDataAccessRuntimeCache.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\DataAccess;
+
+use ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar\MenuItem;
+
+class MenuItemDataAccessRuntimeCache implements MenuItemDataAccessInterface
+{
+    private $categoryCache = null;
+
+    /**
+     * @var MenuItemDataAccessInterface
+     */
+    private $subject;
```

Comment:
```suggestion
    private MenuItemDataAccessInterface $subject;
```

Reply:
Type is applied, also for the cache variable `$categoryCache` (the type is resulted by the assignment below in the code).